Content.Server/Ghost/Roles/Components/GhostRoleMobSpawnerComponent.cs

```diff
@@ -53,6 +50,9 @@ public override bool Take(IPlayerSession session)
             var ghostRoleSystem = EntitySystem.Get<GhostRoleSystem>();
             ghostRoleSystem.GhostRoleInternalCreateMindAndTransfer(session, Owner, mob, this);
 
+            var spawnedEvent = new GhostRoleSpawnerUsedEvent(Owner, mob);
+            _entMan.EventBus.RaiseLocalEvent(mob, spawnedEvent, false);
```

Comment:
В таким случаях, они старый код комментируют, а новый код сверху и снизу помечают комментарием, типа
```suggestion
            // Theta-Ship-Event-Start
            var spawnedEvent = new GhostRoleSpawnerUsedEvent(Owner, mob);
            _entMan.EventBus.RaiseLocalEvent(mob, spawnedEvent, false);
            // Theta-Ship-Event-End
```

Пока у нас нет устоявшейся практики как это лучше делать